javascript/javascript1/week2/booking/booking.js

```diff
@@ -0,0 +1,25 @@
+function getFullName(firstname, surname, useFormalName) {
+    let ourFullName = firstname + " " + surname;
+
+    if (useFormalName === true && firstname === '' && surname === '') {
```

Comment:
Good stuff! Always stick to triple equals when comparing! 

One tip - if the value is already a boolean, you don't need to check if it equals `true` since it already evaluates to true!